src/test/java/org/broadinstitute/hellbender/tools/spark/sv/integration/StructuralVariationDiscoveryPipelineSparkIntegrationTest.java

```diff
@@ -180,14 +180,20 @@ static void svDiscoveryVCFEquivalenceTest(final String generatedVCFPath, final S
                                               final String experimentalOutputPathForNonComplex,
                                               final List<String> attributesToIgnore, final boolean onHDFS) throws Exception {
 
-        List<VariantContext> expectedVcs;
-        try (final VCFFileReader fileReader = new VCFFileReader(new File(expectedVCFPath), false) ) {
-            try (final CloseableIterator<VariantContext> iterator = fileReader.iterator()) {
-                expectedVcs = Utils.stream(iterator).collect(Collectors.toList());
+        final List<VariantContext> expectedVcs;
+        if (expectedVCFPath == null) {
+            expectedVcs = Collections.emptyList();
+        } else {
+            try (final VCFFileReader fileReader = new VCFFileReader(new File(expectedVCFPath), false)) {
```

Comment:
Nitpick, but you can try-with-resources with multiple resources in one statement:
try (final VCFFileReader fileReader = new VCFFileReader(new File(expectedVCFPath), false);
      final CloseableIterator<VariantContext> iterator = fileReader.iterator()) {
...

Reply:
learned something. Thanks!